gateway/1.6/modules/ROOT/pages/policies-included-traffic-management-route.adoc

```diff
@@ -0,0 +1,175 @@
+= Traffic Management for Multiple Upstream Services
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: route, traffic management, upstream service
+
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy Name | Traffic Management for Multiple Upstream Services
+>s|Summary      | Manages API instance traffic to multiple upstream services from a single consumer endpoint
+>s| First Flex Gateway version available | v1.0.0
+.4+>.^s| Returned Status Codes | No return codes exist for this policy.
+[%autowidth.spread,cols="a,a"]
+|===
+
+[NOTE]
+====
+The following information applies only to Flex Gateway running in Local Mode. To configure multiple upstream services for Flex Gateway running in Connected Mode, see xref:api-manager::create-instance-task-flex.adoc#traffic-management[Multiple Upstream Services for Flex Gateway Running in Connected Mode].
```

Comment:
(Leave hard-coded link text for anchor links)

Reply:
See comment above.